backend/game/turn_actions/heal.py

```diff
@@ -0,0 +1,11 @@
+from game.player.player import Player
+import game.values.locations as locations
+
+
+def heal_self_from_dice(player_being_healed: Player, health_points_to_add):
+    if player_being_healed.location is not locations.Locations.TOKYO:
+        player_being_healed.update_health_by(health_points_to_add)
+
+
+def heal_self_no_tokyo_restriction(player_being_healed: Player, health_points_to_add):
```

Comment:
I don't think this method needs to exist?
So far, cards that heal call update_health_by() themselves

Reply:
I didn't think it needed to exist either. Then I second guessed myself and thought I remembered being told we needed it there for some reason I couldn't remember. I'd rather remove it too. will delete